app/views/employee/employee.html.erb

```diff
@@ -3,23 +3,28 @@
 	</head>
 	<body>
```

Comment:
Why do you have html, head and body tags in this file? Remember that these tags appear in application.erb